templates/app-service.json

```
@@ -98,11 +101,22 @@
     }
   },
   "variables": {
+    "appServiceSlotAppSettings": [
+      {
+        "name": "WEBSITE_LOCAL_CACHE_OPTION",
+        "value": "Always"
+      },
+      {
+        "name": "WEBSITE_LOCAL_CACHE_SIZEINMB",
+        "value": "300"
+      }
+    ],
     "useCustomHostname": "[greater(length(parameters('customHostname')), 0)]",
     "appServicePlanId": "[resourceId(parameters('appServicePlanResourceGroup'), 'Microsoft.Web/serverfarms', parameters('appServicePlanName'))]",
     "useAppServiceSlotAppSettings": "[greater(length(parameters('appServiceSlotAppSettings')), 0)]",
     "useAppServiceSlotConnectionStrings": "[greater(length(parameters('appServiceSlotConnectionStrings')), 0)]",
-    "appServiceApiVersion": "2018-11-01"
+    "appServiceApiVersion": "2018-11-01",
+    "appServiceAppSettingsCombined": "[union(parameters('appServiceAppSettings'), variables('appServiceSlotAppSettings'))]"
```

```suggestion
    "appServiceProductionSlotAppSettingsCombined": "[union(parameters('appServiceAppSettings'), variables('appServiceProductionSlotAppSettings'))]"
```
Similarly, could the variable name be changed to make it's clear it's for the Production slot?